asyncio_mqtt/client.py

```diff
@@ -123,6 +124,14 @@ def __init__(
         self._pending_calls_threshold: int = 10
         self._misc_task: Optional["asyncio.Task[None]"] = None
 
+        self._outgoing_calls_sem: Optional[asyncio.Semaphore]
+        if max_concurrent_outgoing_calls:
```

Comment:
Let's use `if max_concurrent_outgoing_calls is not None` instead. Otherwise, the value `0` also takes us to the `else` branch.